build/Common.props

```diff
@@ -10,7 +10,7 @@
     <NetMinimumSupportedVersion>net8.0</NetMinimumSupportedVersion>
     <NetStandardMinimumSupportedVersion>netstandard2.0</NetStandardMinimumSupportedVersion>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
-    <AnalysisLevel>latest-all</AnalysisLevel>
+    <AnalysisLevel>8.0</AnalysisLevel>
```

Comment:
If I don't set the [AnalysisLevel](https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/overview?tabs=net-9#latest-updates) to `8.0`, there will be tons of syntax suggestions showing as errors and failing build: #2313.